node/src/test/kotlin/net/corda/node/services/statemachine/StateMachineManagerTests.kt

```
@@ -111,17 +116,11 @@ class StateMachineManagerTests {
         fiber.actionOnSuspend = {
             throw exceptionDuringSuspend
         }
-        var uncaughtException: Throwable? = null
-        fiber.uncaughtExceptionHandler = UncaughtExceptionHandler { f, e ->
```

No, the exception is not swallowed but rather logged and set into the state machine's result future. This had to be removed because we're no logger allowing exceptions to be thrown unhandled. The relevant change is in `FlowStateMachineImpl#run`.